erpnext/e_commerce/shopping_cart/cart.py

```diff
@@ -501,6 +501,7 @@ def get_party(user=None):
 	contact_name = get_contact_name(user)
```

Comment:
@nextchamp-saqib 

https://github.com/frappe/erpnext/blob/4783e4beee88674aa96eca52321e4b8677a24c17/erpnext/portal/utils.py#L87 

Here we are checking first contact only, we should check all contacts that match email?

Reply:
sharing becaue possibly related to duplicate contacts.

Reply:
Checking only one contact is fine here because on line 517, it just returns the doc if found